src/Home.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useState, useEffect } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+	Content,
+	Container,
+	Form,
+	Section,
+	Heading,
+	Tile,
+	Tag,
+	Notification,
+	Button,
+} from 'react-bulma-components';
+import api from './api.ts';
+
+interface NetItem {
+	title: string
+	containerId: string
+}
+
+interface FormData {
+	spanStart: number | ''
+	spanEnd: number | ''
+	network: number
+}
+
+const Home = ({
+	onModal,
+	nets,
+	setNets,
+	setCurrentDownloadedBlock,
+	isLoading,
+	setLoading,
+}) => {
+	const [formData, setFormData] = useState<FormData>({
+		spanStart: '',
+		spanEnd: '',
+		network: 0,
+	});
+
+  useEffect(() => {
+		if (nets[formData.network].maxBlock === 0) {
+			api('POST', nets[formData.network].rpc, {
+				"jsonrpc": "2.0",
+				"id": 1,
+				"method": "getblockcount",
+				"params": []
+			}).then((res: any) => {
+				const netsTemp = [...nets];
+				netsTemp[formData.network].maxBlock = Math.floor(res.result / 128000) * 128000;
+				setNets(netsTemp);
+
+				if (formData.spanEnd > netsTemp[formData.network].maxBlock) return setFormData({ ...formData, spanEnd: nets[formData.network].maxBlock });
+			}).catch(() => {
+				onModal('failed', 'Object not found: missing or expired');
+			});
+		} else {
+			if (formData.spanEnd > nets[formData.network].maxBlock) return setFormData({ ...formData, spanEnd: nets[formData.network].maxBlock });
+		}
+  },[formData.network]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const fetchBlocksInRange = async () => {
+		if (formData.spanStart === '' || formData.spanEnd === '' || formData.spanEnd < 0) return onModal('failed', 'Insert correct data');
+		if (formData.spanStart < 0 || formData.spanEnd < 0 || formData.spanStart > nets[formData.network].maxBlock || formData.spanEnd > nets[formData.network].maxBlock) return onModal('failed', 'Insert correct borders');
+
+		onModal('loading', formData);
+    setLoading(true);
+		const currentNet: NetItem = nets[formData.network];
+
+		try {
+			const fileHandle = await window.showSaveFilePicker({
+				suggestedName: `snapshot_${nets[formData.network].title}_${formData.spanStart}-${formData.spanEnd}.acc`,
```

Comment:
Please follow sync.ngd.network in naming, it is important for C# node in fact.

Reply:
I do it now, but it's very strange naming: `chain_0.acc`

Reply:
It can be strange, but that's what C# node needs (see https://github.com/neo-project/neo/issues/2998 also).